.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "pybossa/themes/default"]
 	path = pybossa/themes/default
-	url = https://github.com/PyBossa/pybossa-default-theme.git
+	url = https://github.com/bloomberg/pybossa-default-theme.git
```

Comment:
Is it possible to change the git submodule to the original? I cannot merge with pointing to bloomberg submodule

Reply:
@therealmarv , yes, the tests will fail since theme changes are not merged under pybossa/pybossa-default-theme. I will change git submodules to the original, and upon you merge, I think tests would be fine.